test/integration/dynamic-optional-routing/pages/get-static-paths-undefined/[[...slug]].js

```diff
@@ -0,0 +1,25 @@
+export async function getStaticPaths() {
+  return {
+    paths: [
+      {
+        params: {},
```

Comment:
Should this be allowed? IMO it'd be better to require the field be set to `false` or `null`.

Reply:
sure, I just thought it would match nicely with what is passed to `getStaticProps `